bouncer/templates/base.html.jinja2

```diff
@@ -3,18 +3,22 @@
   <head>
     <meta charset=utf-8>
 
+    {% if show_metadata == True %}
```

Comment:
Explicitly checking for boolean values is done with `is` instead of `==`. This should be `{% if show_metadata is True %}`.